src/lib/kernel/LinuxProcess.cc

```diff
@@ -62,11 +62,18 @@ LinuxProcess::LinuxProcess(const std::vector<std::string>& commandLine,
   processImage_ = std::shared_ptr<char>(unwrappedProcImgPtr, free);
 }
 
-LinuxProcess::LinuxProcess(span<char> instructions, ryml::ConstNodeRef config)
+// TODO can this be marked as only usable by test? or is it used by SST??
+LinuxProcess::LinuxProcess(span<const uint8_t> instructions,
+                           ryml::ConstNodeRef config)
     : STACK_SIZE(config["Process-Image"]["Stack-Size"].as<uint64_t>()),
       HEAP_SIZE(config["Process-Image"]["Heap-Size"].as<uint64_t>()) {
   // Set program command string to a relative path of "Default"
-  commandLine_.push_back("Default\0");
+  // TODO need to determine consequences of setting this as absolute and
+  // relative to simeng source directory. Should the default prog be in the
+  // source or copied to the build dir?
+  commandLine_.push_back(SIMENG_SOURCE_DIR "/SimEngDefaultProgram\0");
```

Comment:
I think in any case that the source or build directory moves (which would cause this to break), SimEng will throw an error message saying where it is trying to look so this is probably fine